src/components/Index.vue

```diff
@@ -12,24 +12,20 @@ import suit from '../suit';
 export default {
   provide() {
     this.instantSearchInstance = instantsearch({
-      appId: this.appId,
-      apiKey: this.apiKey,
       indexName: this.indexName,
       routing: this.routing,
       stalledSearchDelay: this.stalledSearchDelay,
+      searchFunction: this.searchFunction,
+      searchParameters: this.searchParameters,
     });
 
     return {
       instantSearchInstance: this.instantSearchInstance,
     };
   },
   props: {
-    apiKey: {
```

Comment:
this change will need to be reflected in the storybooks as well, which used the api key form

Reply:
You've done that right? in `utils.js`?